src/main/java/org/fundacionjala/coding/Ruber/EANValidator.java

```diff
@@ -0,0 +1,58 @@
+package org.fundacionjala.coding.Ruber;
+
+/**
+ * Created by Ruber Cuellar on 2/21/2017.
+ */
+public final class EANValidator {
+
+    private static final int PAR = 1;
+    private static final int IMPAR = 3;
+    private static final int DIVISOR = 10;
+
+    /**
+     * Constructor.
+     */
+    private EANValidator() {
+    }
+    /**
+     * Validate the number if is valid.
+     * @param num the number to validate
+     * @return boolean
+     */
+    public static boolean validate(final String num) {
+        int suma = 0;
+        int[] numbers = convertArray(num);
+
+        for (int i = 0; i < numbers.length - 1; i++) {
+            if (i % 2 == 0) {
```

Comment:
instead of having if else conditional in 5 lines of code.
you can apply a ternary operator here to have one line of code.
```java
suma += i % 2 == 0 ? numbers[i] * PAR : numbers[i] * IMPAR;
```